common/src/main/resources/db/changelog/v2021/rework_user_account_username.sql

```diff
@@ -0,0 +1,19 @@
+--liquibase formatted sql
+--  -------------------------------------------------------------------------------------------------------------------
+
+--changeset wnyffenegger:rework_user_account_username failOnError:true
+
+ALTER TABLE user_account ADD COLUMN organization VARCHAR(255) NOT NULL DEFAULT '';
+
+UPDATE user_account
+SET organization = sponsor.org_name
```

Comment:
I was concerned that if org_name was null, then the set would blow up due to the Null constraint.  I checked in prod and org_name is fully filled for all sponsors.

It would be safer to have a where != null clause but not necessary.